ethcore/engines/authority-round/src/lib.rs

```diff
@@ -1844,9 +1844,10 @@ fn next_step_time_duration(info: StepDurationInfo, time: u64) -> Option<(u64, u6
 		.checked_sub(1)?
 		.checked_sub(info.transition_timestamp)?
 		.checked_div(info.step_duration)?;
+        let time_diff = step_diff.checked_mul(info.step_duration)?;
```

Comment:
Funny whitespace here.

Reply:
Sorry, the tabs always trip me up. Should be fixed now.